libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationEngine.java

```diff
@@ -93,19 +67,20 @@ private void handleRequest(final NewLocationModel newLocationModel) {
     final Location rawLocation = newLocationModel.location();
 
     // Generate a new route progress given the raw location update
```

Comment:
Could we remove the unnecessary comments?

> The best comment is a good name for a method or class.

👀 Code Smells 👉 [Comments](https://refactoring.guru/smells/comments)